setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+with open("requirements.txt", "r") as fh:
+    requirements = fh.read()
+
+setup(
+    name="subtest",
+    version="0.0.0.8",
+    author="SubstraFoundation",
+    author_email="contact@substra.org",
+    description="A distributed learning contributivity package",
+    long_description=long_description,
```

Comment:
Is `long_description` defined elsewhere ?

Reply:
Yes, lines 3-4

Reply:
Ah yes my bad